public_html/app/Models/Post.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace VFramework\Models;
+
+use PDO;
+
+class Post extends AbstractModel
+{
+    /**
+     * @var string
+     */
+    protected $table = 'posts';
+
+    /**
+     * @param $id
+     * @return mixed
+     */
+    public function getById($id)
+    {
+        $query = 'SELECT * FROM posts WHERE id = :id';
```

Comment:
You have function get() in your Abstract class, you that function here $this->get('id'=>$id);